index.html

```diff
@@ -283,50 +283,73 @@
-  themselves to be the subject of a treatment related to this group, even if the
-  entity carrying out the treatment has no way to identify that [=person=].
+A <dfn>user</dfn> (also <dfn data-lt="individual">person</dfn> or <dfn>data subject</dfn>)
+is any natural person.
+
+We define <dfn data-lt="identifier">identifying data</dfn> as any information relating to a
+[=person=] such that any of the following applies:
+
+1. It references this [=person=] directly, for instance through their name,
+  email address, phone number, or other such traditional denotation.
+* It can identify this [=person=] through an arbitrary string or identification
+  number, an online identifier such as an IP address or any identifier attached to a
+  device which this [=person=] may be using.
+* It is about a [=person=]'s location (or of a device that a [=person=] would use) with greater
+  granularity than the city level or a comparably coarse geographic grouping.
+* It contains factors specific to the physical, physiological, genetic, mental,
+  economic, cultural, or social identity of a [=person=].
+* It is derived from other types of data in this list, for instance through hashing.
+* The [=person=] could reasonably be reidentified from a conjunction of this
+  [=data=] with other data.
+* It relates to a group of people such that a [=person=] may be treated in a
+  specific way for being part of the group, even if the entity treating them in this
+  manner has no way to identify that [=person=]. Put differently, an aggregate of [=people=]
+  is identifying if the aggregate is actionable with respect to its members.
+
+We define <dfn data-lt="data">personal data</dfn> as any information that either is
+[=identifying data=] or is related to [=identifying data=].
 
 Data is <dfn>permanently de-identified</dfn> when there exists a high level of confidence
-that no human subject of the data can be identified, directly or indirectly (e.g., via
-association with an identifier, user agent, or device), by that data alone or in
-combination with other retained or available information, including as being part of a
-group. Note that further considerations relating to groups are covered in the
+that no [=person=] described by the data can be identified, directly or indirectly
+(e.g. via association with an [=identifier=], user agent, or device), by that data alone or in
+combination with other available information, including as being part of a group. Note
+that further considerations relating to groups are covered in the
 <a href="#collective">Collective Issues in Privacy</a> section.
 
-Data is <dfn data-lt="pseudonymous data|pseudonymity">pseudonymous</dfn> when:
+Data is <dfn data-lt="pseudonymous data|pseudonymity">pseudonymous</dfn> when there are
+strict controls that prevent the reidentification of [=people=] described by the data
+except for a well-defined set of [=purposes=]. There are two cases in which [=data=]
```

Comment:
Would it make sense to limit the section on pseudonymity to just this first sentence? I feel like it's not really correct to divide the two cases here, and the discussion of how third-parties can use the data isn't right either, but instead of nitpicking that, maybe we can delete it.